package.json

```diff
@@ -359,6 +359,11 @@
                 "title": "Connect Zeplin Components to Component in Configuration",
                 "category": "Zeplin"
             },
+            {
+                "command": "zeplin.migrateZeplinComponents",
+                "title": "Migrate Zeplin Component names to ids in Configuration",
```

Comment:
We should better keep "in Configuration" here like the commands above this one to indicate (a little) it's related to CoCo feature because CoCo is not the only feature in the extension.

Reply:
"Component" word starts with Uppercase in command titles. Shall we keep it for consistency?

Reply:
Also "Zeplin Component names" to "Component ids" may be confusing because we have "components" and "zeplin components". I may be misunderstood. It would be better keep it the way it is currently. Another option is "Migrate Zeplin Component names to Zeplin Component ids" which is a little verbose. "Migrate Zeplin Component names to ids in Configuration" seems like the best shot here.

Reply:
I'd suggest uppercasing ID in all the copies: "…names to IDs in…"